actionpack/lib/action_dispatch/railtie.rb

```diff
@@ -32,7 +32,6 @@ class Railtie < Rails::Railtie # :nodoc:
       "X-Frame-Options" => "SAMEORIGIN",
       "X-XSS-Protection" => "1; mode=block",
       "X-Content-Type-Options" => "nosniff",
-      "X-Download-Options" => "noopen",
```

Comment:
This probably exists for compatibility with `load_defaults "7.0"`.

Reply:
But these all are default headers, right? Also, the test for this case is removed from https://github.com/rails/rails/pull/43968/commits/486be06ccbec8b66e6d674f57fc957eb7ba7e011#:~:text=assert_equal(%22noopen%22%2C%20resp.headers%5B%22X%2DDownload%2DOptions%22%5D)

Reply:
It is the default version for <= 6.1.
For 7.0 we override the "X-XSS-Protection" from "1; mode=block" to "0" (visible here as well).
For 7.1 we remove "X-Download-Options".

So someone upgrading from 6.1 to 7.0, would still see "1; mode=block" unless they change the framework default.